new.go

```diff
@@ -37,12 +37,11 @@ func (n *newCmd) curlCommand(e *parsecli.Env, app *parsecli.App) string {
  -H "X-Parse-REST-API-Key: %s" \
  -H "Content-Type: application/json" \
  -d %s \
- %s/1/functions/hello
```

Comment:
:-1:

Functions endpoint is `/1/functions/hello`. (https://github.com/back4app/parse-cli-server/blob/master/src/HooksRouter.js#L29)

The back4app endpoint is the default `e.Server` value, but it is possible to override it with env var `PARSE_SERVER`.